### PR TITLE
Externals for libhidapi, libusb

### PR DESCRIPTION
**Pull request by pmunts**
Added libhidapi and libusb

### CODE REVIEW

File: index/li/libhidapi/libhidapi-external.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description = "A simple library for communicating with USB and Bluetooth HID devices"
+name = "libhidapi"
+
+maintainers = ["Philip Munts <phil@munts.net>"]
+maintainers-logins = ["pmunts"]
+
+[[external]]
+kind = "system"
+[external.origin."case(distribution)"]
+"debian|ubuntu" = ["libhidapi-dev"]
```
Comment (onox) [inline index/li/libhidapi/libhidapi-external.toml:11]: You could add support for Arch Linux as well with `arch = ["hidapi"]`

File: index/li/libhidapi/libhidapi-external.toml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+description = "A simple library for communicating with USB and Bluetooth HID devices"
+name = "libhidapi"
```
Comment (onox) [inline index/li/libhidapi/libhidapi-external.toml:2]: `licenses = "BSD-3-Clause OR GPL-3.0-only OR custom-hidapi"` could be added. See https://github.com/libusb/hidapi/blob/master/LICENSE.txt

File: index/li/libusb/libusb-external.toml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+description = "A cross-platform library to access USB devices"
+name = "libusb"
```
Comment (onox) [inline index/li/libusb/libusb-external.toml:2]: `licenses = "LGPL-2.1-or-later"`. See https://github.com/libusb/libusb

File: index/li/libusb/libusb-external.toml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+maintainers = ["Philip Munts <phil@munts.net>"]
+maintainers-logins = ["pmunts"]
+
+[[external]]
+kind = "system"
+[external.origin."case(distribution)"]
+"debian|ubuntu" = ["libusb-1.0-0-dev"]
```
Comment (onox) [inline index/li/libusb/libusb-external.toml:11]: Support Arch Linux with `arch = ["libusb"]`

**Comment by mosteo** [comment]
Thanks for the suggestions, @onox. I took the liberty of pushing the necessary changes and will merge as soon as the checks pass again.

**Review by mosteo — APPROVED** [review]

**Pull request merged**